src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -947,7 +947,7 @@ public class SpacingRuleLayouts : EnumAsInt<SpacingRuleLayout>
         public SpacingRuleLayouts(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts) { }
     }
 
-    [NodeName("Element Types")]
+    [NodeName("Element Classes")]
```

Comment:
This class is subclass of AllChildrenOfType<Element>, and can't modify its output name.
The output name is set in DynamoCore - https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/Enum.cs#L65 .
So I think I need submit a update here.

Reply:
![ElementClass](https://user-images.githubusercontent.com/33445445/96690351-232c1800-13b6-11eb-885e-97cea8be8e4a.png)
Hi @ShengxiZhang , with this PR, the results as the screenshot shown. We can't rename the output name "Type" on DynamoRevit side.